cypress-tests/cypress/e2e/configs/Payment/Stripe.js

```diff
@@ -314,6 +314,15 @@ export const connectorDetails = {
           amount_received: 6000,
         },
       },
+      ResponseCustom: {
+        status: 200,
+        body: {
+          status: "succeeded",
+          amount: 8000,
+          amount_capturable: 0,
+          amount_received: 8000,
+        },
+      },
```

Comment:
this ain't gonna work. support for `ResponseCustom` is broken / dependent on spec file. 

cc: @likhinbopanna

Reply:
and incremental tests are failing for stripe:

<img width="897" height="199" alt="image" src="https://github.com/user-attachments/assets/6e2bfdb6-7fec-4668-bb91-c7ff69eeff0d" />